scripts/compression_scripts.sh

```
@@ -0,0 +1,11 @@
+# To convert a set of webp images to a smaller size and quality
+for f in *.webp ; convert "$f" -coalesce -quality 50 -resize 128x -define webp:lossless=false,method=6 "compressed_${f%}" ;
```

Shellcheck (0.10.0)

[error] Couldn't parse this for loop. Fix to allow more checks.

(SC1073)